code/controllers/subsystem/ticker.dm

```diff
@@ -271,11 +271,15 @@ SUBSYSTEM_DEF(ticker)
 		cb.InvokeAsync()
 	LAZYCLEARLIST(round_start_events)
 
+<<<<<<< HEAD
 	SEND_SIGNAL(src, COMSIG_TICKER_ROUND_STARTING)
 	real_round_start_time = world.timeofday //SKYRAT EDIT ADDITION
+=======
+	round_start_time = world.time //otherwise round_start_time would be 0 for the signals
+	SEND_SIGNAL(src, COMSIG_TICKER_ROUND_STARTING, world.time)
+>>>>>>> 1ac2779ef89 (Bileworm Health Patch: Evolutionary Leap, Vileworms, Proximity Spawning Rules, Better Loot (#70321))
```

Comment:
```suggestion
	round_start_time = world.time //otherwise round_start_time would be 0 for the signals
	SEND_SIGNAL(src, COMSIG_TICKER_ROUND_STARTING, world.time)
	real_round_start_time = world.timeofday //SKYRAT EDIT ADDITION
```